gradle.properties

```diff
@@ -14,12 +14,8 @@ org.gradle.jvmargs=-Xmx6656m -Dfile.encoding=UTF-8
 org.gradle.parallel=true
 
 android.useAndroidX=true
-kapt.incremental.apt=true
```

Comment:
Why is this line deleted?

Reply:
Some configuration has been deprecated or removed when updating Kotlin or Gradle. In this particular case, kapt was not even used.